packages/router/tests-setup.js

```diff
@@ -1,5 +1,8 @@
+import fromAsync from 'array-from-async';
 import { vi } from 'vitest';
 
+Array.fromAsync = fromAsync;
```

Comment:
Since we're not actually running Chrome, which has already implemented `Array.fromAsync`, we need to polyfill it in our test environment